ui/analyse/src/keyboard.ts

```diff
@@ -85,6 +85,28 @@ export const bind = (ctrl: AnalyseCtrl) => {
     };
     keyToMousedown('d', '.study__buttons .comments');
     keyToMousedown('g', '.study__buttons .glyphs');
+
+    // navigation for next and prev chapters
+    const study = ctrl.study;
+    kbd
+      .bind('n', () => {
+        const chapters = study.chapters.list();
+        let next = null;
+        for (const i in chapters) {
+          if (chapters[i].id === study.currentChapter().id)
+            next = chapters[parseInt(i) + 1];
```

Comment:
You can just do `study.setChapter(chapters[parseInt(i) + 1].id)` here and then `break` or `return`. No need for the additional variable.